03-kogito-travel-agency/pom.xml

```diff
@@ -10,59 +10,18 @@
   <properties>
     <surefire-plugin.version>2.22.0</surefire-plugin.version>
     <maven.compiler.target>1.8</maven.compiler.target>
-    <kogito.version>0.4.0</kogito.version>
+    <kogito.version>0.8.0</kogito.version>
     <maven.compiler.source>1.8</maven.compiler.source>
-    <quarkus.version>0.23.2</quarkus.version>
+    <quarkus.version>1.3.0.Alpha2</quarkus.version>
```

Comment:
do we need this if we use the bom? (same question for all occurrencies)

Reply:
not mandatory, but seems like a good practice to me.

Reply:
but can't you just use `${version.io.quarkus}` ? (not sure it actually works, that's why I am asking)

Reply:
not really, properties don't get imported into another pom AFAIK. Perhaps we could define the plugin using pluginmanagement, but not sure either if that would work, as it works for regular dependencies.